full/build.gradle

```diff
@@ -46,8 +46,11 @@ shadowJar {
 
 dependencies {
     implementation project(":common")
-    apt project(":processor")
+    testImplementation project(":common").sourceSets.test.output
     testImplementation project(':test-utils')
+    testImplementation project(":core")
```

Comment:
Unfortunately we need to add this for the moment: `full->test` depending on `core`. I don't know whether we are going to be able to take out this dependency easily.